src/Rector/Deprecation/DatetimeDatetimeStorageFormatRector.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation;
+
+use DrupalRector\Rector\Deprecation\Base\ConstantToClassConstantBase;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * Replaces deprecated DATETIME_DATETIME_STORAGE_FORMAT constant use.
+ *
+ * See https://www.drupal.org/node/2912980 for change record.
+ *
+ * What is covered:
+ * - See `ConstantToClassConstantBase.php`
+ *
+ * Improvement opportunities
+ *  - See `ConstantToClassConstantBase.php`
+ */
+final class DatetimeDatetimeStorageFormatRector extends ConstantToClassConstantBase
+{
+  protected $deprecatedConstant = 'DATETIME_DATETIME_STORAGE_FORMAT';
+
+  protected $constantFullyQualifiedClassName = 'Drupal\datetime\Plugin\Field\FieldType\DateTimeItemInterface';
+
+  protected $constant = 'DATETIME_STORAGE_FORMAT';
+
+
+  /**
+   * @inheritdoc
+   */
+  public function getDefinition(): RectorDefinition
+  {
+    return new RectorDefinition('Fixes deprecated DATETIME_DATETIME_STORAGE_FORMAT use',[
+      new CodeSample(
+        <<<'CODE_BEFORE'
+$date = new DrupalDateTime('now', new \DateTimezone('America/Los_Angeles'));
```

Comment:
Here.

Reply:
This is used only for generating documentation as far as I know, but it would be good if it was valid. Do we need to use the fully qualified class name?

Reply:
Fixed in cd8af284dc7cd96a22e08025f3b351dd3ac387f7